FischerLiu.md

```diff
@@ -1,18 +1,45 @@
-## 10/26 - 10/30 ##
+## 11/09 - 11/13 ##
 
 ### This Week ###
 * [Fling Player]
 
-  - [REVIEW] Bug 1217287 - [Stingray][fling-player][TV][2.5] (TV side) Should have the unit test for the connector.js
+  - [Updating] Bug 1222364 - [Stingray][fling-player][TV][2.5] Implement for the specs updates
+
+    - Updating for the code review
+
+* [Send video to TV App]
+
+  - [REVIEW] Bug 1220584 - [TV] FTE for Casting video and website to TV
+
+    - Updated for the code review
+
+* [Marketplace FTE]
+
+  - [FIXED] Bug 1223346 - [TV] FTE for Marketplace
 
-* [Casting Promotion App]
+    - Complete the visual, script controlling, l10n and file structure
+      and give it to Joseph to integrate into the TV Marketplace App in the future.
 
-  - [FIXED] Bug 1221859 - [TV] Website for casting-video and website tutorial
+* [App Usage Metric FTE]
 
-  - [WIP] Bug 1221857 - [TV] Implement the website visual specs for casting tutorial
+  - [WIP] Bug 1220588 - [TV] FTE for usage matrics
 
 ### Last Week ###
+* [Fling Player]
+
+  - [REVIEW] Bug 1217287 - [Stingray][fling-player][TV][2.5] (TV side) Should have the unit test for the connector.js
+
+    - Reviewing is done. Require updates. Will update after other 1st priority works are done.
+
+* [Send video to TV App]
+
+  - [FIXED] Bug 1221857 - [TV] Implement the website visual specs for casting tutorial
+
+* [Other]
 
-* [Casting Promotion App]
+  - [FIXED] Bug 1222909 - Update FTEWizard
 
-  - [FIXED] Bug 1221042 - Add the primary style for the 'rounded-corner-text' smart button type
\ No newline at end of file
+### 2015 Q4 Goal ###
+1. Improve the fling player UI and User experience, like more interactions with the controlling device.
+2. Improve the file structure and the tests for apps including Fling Player, FTE-related apps
+3. Fix other minor issues found before including update smart-button and update the dependency from smart-components to gaia-components
```

Comment:
I think we should sit-down and discuss these one by one - I'm not sure what you are talking about. Arrange a meeting with me this week. Thanks!